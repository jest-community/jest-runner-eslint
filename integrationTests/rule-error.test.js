const runJest = require('./runJest');

it('Correctly prints the error message from a failing ESLint rule', async () => {
  expect(await runJest('rule-error')).toMatchSnapshot();
});
