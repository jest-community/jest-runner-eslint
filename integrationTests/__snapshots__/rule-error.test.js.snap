// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Correctly prints the error message from a failing ESLint rule 1`] = `
"FAIL integrationTests/__fixtures__/rule-error/__eslint__/one.js
Cannot read property 'cause' of undefined
Occurred while linting /mocked-path-to-jest-runner-mocha/integrationTests/__fixtures__/rule-error/__eslint__/one.js:1
FAIL integrationTests/__fixtures__/rule-error/__eslint__/two.js
Cannot read property 'cause' of undefined
Occurred while linting /mocked-path-to-jest-runner-mocha/integrationTests/__fixtures__/rule-error/__eslint__/two.js:1
Test Suites: 2 failed, 2 total
Tests:       2 failed, 2 total
Snapshots:   0 total
Time:
Ran all test suites.
"
`;
